README.md

```diff
@@ -104,3 +104,12 @@ Lnd data and logs are written to the following locations in production:
 * **Linux:** `~/.config/lightning-app/lnd`
 * **OSX:** `~/Library/Application Support/lightning-app/lnd`
 * **Windows:** `%USERPROFILE%\AppData\Roaming\lightning-app\lnd`
+
+### Connect to a remote lnd node
+1. In your remote node's `lnd.conf`, set `rpclisten=0.0.0.0:10006` and `tlsextraip=<insert your node's IP address here>`, then restart the node.
+
+2. Copy the `tls.cert` and `tls.key` from your remote node's `.lnd` into the appropriate folder for your platform (specified in the section above).
```

Comment:
Will I overwrite my `tls.cert` and `tls.key`? Would that prevent the app from running in bundled mode?

Reply:
Yeah it's a problem. I think I'm going to change it to accept CLI arguments for the path to the certs and the macaroons instead of telling the user to put them in a specific place. LMK if that works.

Reply:
Also, we should add this section in the read above `### Lnd data and logs` since it goes together with the different modes of operation.

Reply:
Cool! Ok then it shouldn't be an issue 👍

Reply:
ok sounds good, fixed the order of the README